### PR TITLE
Fixes to handlers and fixing some small bugs.

### PR DESCRIPTION
**Pull request by pseeth**
This PR fixes some odds and ends I ran into recently. I'll do a review of it myself inline. 

Summary of changes:

- Removed the output norm in DualPath
- Added a gated sigmoid activation function to Embedding
- Added some regularization of stuff.
- Fixed the tensorboard handler so it doesn't save a new file for every epoch.
- Fixed the time elapsed handler so it doesn't wrap around at a day.

### CODE REVIEW

File: nussl/ml/networks/modules/blocks.py [hunk 1]
```diff
@@ -324,7 +324,7 @@ class Embedding(nn.Module):
         embedding_size (int): Dimensionality of embedding.
         
         activation (list of str): Activation functions to be applied. Options 
-          are 'sigmoid', 'tanh', 'softmax', 'relu'. Unit normalization can be applied by 
+          are 'sigmoid', 'tanh', 'softmax', 'relu', 'gate'. Unit normalization can be applied by
```
Comment (ethman) [inline nussl/ml/networks/modules/blocks.py:327]: I'd rename this to `gated_tanh` to be more descriptive.

**Review by ethman — APPROVED** [review]
LGTM

File: nussl/ml/networks/modules/blocks.py (hunk 1 above)
Comment (pseeth) [reply nussl/ml/networks/modules/blocks.py:327]: Done, let's just wait on tests.

**Comment by pseeth** [comment]
Merging this into master after tests pass. I caught one more issue - if ffmpeg isn't installed we throw a warning, but warnings hasn't been imported yet in `__init__.py` sooo everything crashes. Classic.
